testRes/json/ProceduralFairings.json

```diff
@@ -0,0 +1,842 @@
+[
```

Comment:
I don't think we should for two reasons.

1) It's used for debugging and it really does need to be human readable.
2) It's a literal copy of what github produces.